audbackend/core/artifactory.py

```diff
@@ -109,15 +112,39 @@ def _glob(
 
     def _ls(
             self,
-            path: str,
+            folder: str,
```

Comment:
We could set `''` as default value, then it would simply show all files on the backend if no `folder` is specified.

Reply:
OK, then I would set `'/'` as default.

Reply:
done